draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -755,31 +755,34 @@ An elliptic curve E is specified by a cubic equation in two variables and a
 finite field F. An elliptic curve equation takes one of several standard forms,
 including (but not limited to) Weierstrass, Montgomery, and Edwards.
 
-The curve E forms an algebraic group whose elements are the points (x, y)
-satisfying the curve equation, where x and y are elements of F. This group
-has order n, meaning that there are n distinct points (x, y). In general,
-security of cryptographic primitives requires using a group of prime order.
-However, not all elliptic curves induce groups of prime order: most elliptic
-curves have order n = h * r, where r is a large prime and h is an
-integer called the cofactor. Thus, for cryptographic applications a hash
-function to the curve E should return points in the subgroup of order r. For
-a point not in the prime-order subgroup, the process of mapping to a point
-in the prime-order subgroup is called clearing the cofactor; we discuss this
-process in {{cofactor-clearing}}.
-
-Certain encoding functions restrict the form of the curve equation, the characteristic
-of the field, and/or the parameters of the curve. For each encoding,
-this document lists the relevant restrictions.
+The curve E forms an algebraic group, denoted as E(F), whose elements are the
+points (x, y) satisfying the curve equation, where x and y are elements of F.
+This group has order n, meaning that there are n distinct points.
+
+In general, security of cryptographic primitives requires a group G of
```

Comment:
@kwantam does it refer to the generator in the related work? If so, perhaps we could unify around `P` (or whatever we use in the rest of the document)?

Reply:
My reasoning to introduce G is because for a suite for pairing curves will refer to the same curve but different subgroups, say G1 and G2. 

I ack that, In some cases, it is more convenient to use only one character, and in others, a verbose text is better. However, G is a general letter used for groups, and we are proposing a way to hashing into a group. 

Finally, G also clashes with generator of curves, but in this draft no generators are used.

Reply:
You're right, it does make sense to use G, G1, G2 as you describe. :+1:

There is one conflicting use of `G` as a generator in the relwork section on the master branch. Also, the commit 1de5f6ebba134de9584e0d2fe8954ecd88a0e4ab introduced a conflict with the master branch, because it edits the "Try-And-Increment" appendix, which #122 removed.

It should be a quick fix to remove that piece of the commit (it's the very last line of the diff). @armfazh, when you fix the conflict, can you please at the same time rebase onto master and rename `G` to `P` in the relwork section? The only occurrence is in the second-last paragraph (the one that starts "Brier et al. ...")